portfolio/src/main/webapp/index.html

```diff
@@ -170,30 +170,30 @@ <h1>Contact Me</h1>
           Feel free to write me an email at zoe.barth@gmail.com or leave a
           comment below:
         </p>
-        <form action="/data" method="POST">
+        <form id="comment-form">
           <textarea id="comment" name="comment" class="form-control" rows="5"> </textarea>
           <br />
-          <input type="submit" class="btn btn-dark" />
+          <button type="submit" class="btn btn-dark">Submit</button>
         </form>
         <br />
-        <p> Here are the comments: </p>
-        <div id="comments-container"> </div>
-        <p> Number of comments to display:
+        <p> Comments displayed:
         <select name="limit" id="limit" class="selectpicker" onchange="fetchContent()"> 
           <option value="5">5</option>
           <option value="10">10</option>
           <option value="25">25</option>
           <option value="all">All</option>
         </select>
+        <div id="comments-container"> </div>
         <br />
         <button class="btn btn-danger btn-sm" onclick="deleteComments()">Delete all comments</button>
       </div>
     </section>
 
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
I'm confused, this looks like you have 2 copies of jquery, jquery 3.2.1 & jquery 3.5.1. Do you need to load both? and if so, why?